Адаптивная сетка

licos87 opened the pull request:
---
:mortar_board: [Адаптивная сетка](https://up.htmlacademy.ru/adaptive-individual/1/user/323345/tasks/7)

:boom: https://htmlacademy-adaptive.github.io/323345-cat-energy-1/4/

licos87 committed:
- modified package-lock.json

licos87 committed:
Merge branch 'master' of github.com:htmlacademy-adaptive/323345-cat-energy-1

licos87 committed:
- адаптация страницы index.html до блока how-work

licos87 committed:
- добавляет иконки

licos87 committed:
- адаптация live-model

licos87 committed:
- завершение адаптации страницы index.html

licos87 committed:
- завершает адаптацию страницы form.html

licos87 committed:
- завершает адаптацию страницы catalog.html

licos87 committed:
- исправляет валидацию БЕМ и последующие ошибки

licos87 committed:
- убирает пустые строки в конце файла

licos87 committed:
- перезаливка

licos87 committed:
- удаление изображений

licos87 committed:
- перезагрузка изображений

licos87 committed:
- меняет пути шрифтов

licos87 committed:
- добавляет резину на страницы

licos87 committed:
- правит изображение кошки
- правит положение адресса и карту

licos87 committed:
- добавляет работу кнопки toggle через js

licos87 committed:
- добавляет анимацию на бургер
- добавляет анимацию на формы
- переименовывает файлы
- добавляет гугл карту

vadimrahmanov commented on source/js/script.js:
```diff
@@ -0,0 +1,14 @@
+let navMain = document.querySelector('.main-menu');
+let navToggle = document.querySelector('.main-menu__toggle');
```
если ты не переназначаешь переменную, то можно использовать `const`

vadimrahmanov commented on source/sass/blocks/card.scss:
```diff
@@ -69,3 +69,72 @@
+    flex-direction: column;
+
+    &--more {
+      width: 170px;
```
Я уже говорил про фиксирование ширины. Здесь лучше добиться с помощью паддингов, ну или раз ты уж хочешь ограничить с помощью такого способа, то используй хотя бы max-width.

vadimrahmanov commented on source/sass/blocks/catalog.scss:
```diff
@@ -59,3 +59,61 @@
+  }
+
+  .catalog__item:not(:last-child) {
+    background: linear-gradient(180deg, #ffffff 0%, #ffffff 28%, #f2f2f2 28%, #f2f2f2 100%);
```
здесь цвета можно использовать из переменных

vadimrahmanov commented on source/sass/blocks/contacts.scss:
```diff
@@ -5,14 +5,16 @@
 .contacts__wrap {
-  padding: 0 20px;
   display: grid;
   grid-template-columns: 1fr 1fr;
```
здесь кстати тоже уместно использовать repeat(2, 1fr)

vadimrahmanov commented on source/sass/blocks/contacts.scss:
```diff
@@ -21,14 +23,62 @@
 }
 
 .contacts__address {
+  width: 140px;
```
про фиксирование ширины писал выше.

vadimrahmanov commented on source/form.html:
Поля не должны быть красными ещё до того как пользователь начинает ввод

vadimrahmanov commented on source/catalog.html:
Не проходит валидацию БЭМ

vadimrahmanov commented on source/form.html:
Не проходит валидацию БЭМ

licos87 committed:
- заменяет let на const  в JS
- исправляет валидацию БЕМ
- исправляет валидацию форм
- убирает фиксированную ширину
- заменяет цвета на переменные

licos87 committed:
- меняет width на max-width в заголовке карточки товара
